environment.yml

```diff
@@ -40,7 +40,7 @@ dependencies:
 - pycrypto=2.6.1
 - pytest=5.3.5
 - python=3.7.6
-- pyyaml=3.13
+- pyyaml=5.2
```

Comment:
Out of curiosity, is there a reason why the latest 5.4.1 (https://pypi.org/project/PyYAML/5.4.1/) cannot be used? For example, would it conflict with other dependencies in the image (and therefore crash the CircleCI build)?